src/OpenStudy/relations.jl

```diff
@@ -119,6 +120,74 @@ function _get_relation(source::String, target::String, relation_type::RelationTy
     return _RELATIONS[source][(target, relation_type)]
 end
 
+function _get_target_index_from_relation(data::Data, source::String, target::String, source_index::Integer, relation_attribute::String)
+    source_element = data.raw[source][source_index]
+    target_index = _get_index(data.data_index,source_element[relation_attribute])
+    return target_index[2]
```

Comment:
Why `target_index[2]`? It would be nice to have some comments

Reply:
I noticed that `_get_index()` doesn't return only an element's index. It returns a tuple `(CollectionName, Index)`. That's why I had to access the second item... I could leave it commented on the code or we could think of a new approach.